Assets/Shaders/Fog.shader

```diff
@@ -35,7 +35,7 @@ Shader "Kaima/Depth/Fog"
 			sampler2D _MainTex;
 			float4 _MainTex_TexelSize;
 			sampler2D _CameraDepthTexture;
-			fixed4 _FogColor;
+			float4 _FogColor;
```

Comment:
Changing from fixed to float precision just because I feel in my heart that we don't need to optimize the precision here 🤷